### PR TITLE
fix: pinning permissions

### PR DESCRIPTION
**Pull request by yasmin-e**
Closes: #5375 

### Description:
- [x] New **PinnedItems** ability object
- [x] As an `Admin`, `editor`, or `developer`, I can **pin/unpin** and **re-order** items
- [x] As a `Member`, `Viewer`, or `Interactive viewer`, I can only view pinned items

### CODE REVIEW

**Review by TuringLovesDeathMetal — APPROVED** [review]
Nice! Looks good to me 🚀

File: packages/frontend/src/components/PinnedItemsPanel/index.tsx [hunk 1]
```diff
@@ -22,16 +22,17 @@ const PinnedItemsPanel: FC<Props> = ({
...
 }) => {
     const { user } = useApp();
-    const userCanUpdateProject = user.data?.ability.can(
-        'update',
-        subject('Project', { organizationUuid, projectUuid }),
+    const userCanManagePinnedItems = user.data?.ability.can(
+        'manage',
+        subject('PinnedItems', { organizationUuid, projectUuid }),
     );
+    console.log({ userCanManagePinnedItems, user });
```
Comment (joaoviana) [inline packages/frontend/src/components/PinnedItemsPanel/index.tsx:29]: could you remove this pls?

**Review by joaoviana — CHANGES_REQUESTED** [review]
only one small thing, then good to go!

File: packages/frontend/src/components/common/ResourceView/ResourceActionMenu.tsx [hunk 2]
```diff
@@ -151,8 +151,8 @@ const ResourceViewActionMenu: FC<ResourceViewActionMenuProps> = ({
                 )}
 
                 {user.data?.ability.can(
-                    'update',
-                    subject('Project', { organizationUuid, projectUuid }),
+                    'manage',
+                    subject('PinnedItems', { organizationUuid, projectUuid }),
```
Comment (ZeRego) [inline packages/frontend/src/components/common/ResourceView/ResourceActionMenu.tsx:155]: We need the same fix on packages/frontend/src/components/Explorer/SpaceBrowser/SpaceBrowserMenu.tsx

**Review by ZeRego — APPROVED** [review]

**Review by joaoviana — APPROVED** [review]

**Pull request merged**